packages/flutter_tools/lib/src/android/gradle.dart

```diff
@@ -503,22 +506,29 @@ Future<void> buildGradleApp({
   );
 
   // Call size analyzer if --analyze-size flag was provided.
```

Comment:
What does the command to generate the code size analysis look like now? does it still use a --analyze-size flag?

Reply:
Yes